entrypoint.sh

```diff
@@ -80,20 +80,21 @@ do
       /templates/vhost.sample.conf > "$dest"
   upstreamId=$((upstreamId+1))
 
-  #prepare the letsencrypt command arguments
-  letscmd="$letscmd --domain $t "
+  # prepare the letsencrypt command arguments, but skip if cached
+  if [ ! -f /etc/letsencrypt/live/"${t}"/fullchain.pem ]; then
+    letscmd="$letscmd --domain $t "
+  fi
```

Comment:
This would not work since we need to generate 1 certificate for all the domains (to avoid the letsencrypt 5-certs-a-week limit). Thus, only one cert is stored, so the per-domain subfolder make no sense.